story/Clara-Princess-Aisha.txt

```diff
@@ -0,0 +1,9 @@
+I am standing in Clara's study. She is reading a letter from a Princess Aisha. She explains to me that Princess Aisha is a daughter of one of Mars's most powerful rulers,
+and that she (Clara) is in secret correspondence with her. What could this be about? Why is Clara, an ordinary woman in an ordinary town in Ontario province, corresponding
+with a Princess from Mars? What is the secret? What does Clara do? To these questions I can only hazard a guess. Maybe later I will know her well enough to ask them? She
+tells me the Princess is worried about her father, and that the consequences could even be felt here on Earth, in Canada or possible even in my home country, the United Kingdom.
+She can tell me no more, she says. She puts the letter in a locked drawer. We talk about Martian politics, about speculation that some of the Martian Kings and Queens are planning
+to interfere in elections on Earth, maybe in the United States. Some people think this is a conspiracy theory. I ask Clara what she thinks. She says it probably is just a conspiracy 
+theory, but her face tells me something else. I think she thinks it's really going to happen. Maybe she even knows about the plot. Thoughts whizz through my head. What would
+POTUS think if she knew that a Canadian was party to a plot to interfere in American elections? If it gets out will there be war? Is it safe for me to continue my visit? I tell
+myself this is all speculation, it probably is just a conspiracy theory, I should stop worrying. But...
```

Comment:
80 column width
```suggestion
I am standing in Clara's study. She is reading a letter from a Princess
Aisha. She explains to me that Princess Aisha is a daughter of one of Mars's
most powerful rulers, and that she (Clara) is in secret correspondence with
her. What could this be about? Why is Clara, an ordinary woman in an ordinary
town in Ontario province, corresponding with a Princess from Mars? What
is the secret? What does Clara do? To these questions I can only hazard a
guess. Maybe later I will know her well enough to ask them? She tells me the
Princess is worried about her father, and that the consequences could even
be felt here on Earth, in Canada or possible even in my home country, the
United Kingdom.  She can tell me no more, she says. She puts the letter in a
locked drawer. We talk about Martian politics, about speculation that some of
the Martian Kings and Queens are planning to interfere in elections on Earth,
maybe in the United States. Some people think this is a conspiracy theory. I
ask Clara what she thinks. She says it probably is just a conspiracy theory,
but her face tells me something else. I think she thinks it's really going
to happen. Maybe she even knows about the plot. Thoughts whizz through my
head. What would POTUS think if she knew that a Canadian was party to a plot
to interfere in American elections? If it gets out will there be war? Is
it safe for me to continue my visit? I tell myself this is all speculation,
it probably is just a conspiracy theory, I should stop worrying. But...
```

Reply:
I've changed it to a markdown which I think works better :)